gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Jul 28 17:46:00 CEST 2017
+#Tue Aug 08 00:31:49 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
```

Comment:
hm, this is what you get by `./gradlew wrapper --gradle-version=4.1` so should be correct. 

Here's the Gradle Change log, feel free to merge https://docs.gradle.org/current/release-notes.html